src/briefcase/platforms/windows/msi.py

```diff
@@ -144,80 +143,79 @@ def package_app(self, app: BaseConfig, **kwargs):
 
         :param app: The application to build
         """
-        self.logger.info()
         self.logger.info("Building MSI...", prefix=app.app_name)
 
         try:
             self.logger.info()
```

Comment:
On a theme of consistency - are these `logger.info()`s still needed?

Reply:
I actually struggled with these empty `info()` calls.....because at least one of them always results in output to the terminal.

Here's the terminal's final state without the `info()` calls:

```
(venv-3.9) PS C:\Users\Russell\tmp\beeware-tutorial\helloworld> briefcase package

[helloworld] Building MSI...
Compiling application manifest... done
helloworld.wxs
helloworld-manifest.wxs
Compiling application installer... done
Linking application installer... done

[helloworld] Packaged windows\Hello World-0.0.1.msi
```

That print out concerning wxs files is related to installer compilation....but it appears related to manifest compilation.

I talk about this more in my comment.